contrib/bookmark-evernote.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#basic newsboat bookmark plugin for evernote using geeknote: https://github.com/pipakin/geeknote
+
+url="$1"
+title="$2"
+description="$3"
+feed_title="$4"
+
+content="${url}"$'\n'"${description}"$'\n'${feed_title}
```

Comment:
I think `${feed_title}` should be in double quotes, too.

Also, what is this `$'\n'` syntax? I don't think I ever saw it before.

Reply:
You're right, will add in the double quotes. Weird newline syntax is because of an issue with geeknote where it wasn't taking the newline correctly.